connectors/es/sink.py

```diff
@@ -309,18 +313,27 @@ async def _run(self):
                         self._batch_bulk,
                         copy.copy(batch),
                         copy.copy(stats),
-                    )
+                    ),
+                    result_callback=functools.partial(self.batch_bulk_callback),
```

Comment:
Alternatively, we could set `self.sink_error` inside a try/except in `_batch_bulk()`. That's how the Extractor works today.

Reply:
That's what we landed on